### PR TITLE
[DOCFIX] Improve docker documentation

### PR DESCRIPTION
**Pull request by apc999**

### CODE REVIEW

File: docs/en/deploy/Running-Alluxio-On-Docker.md [hunk 1]
```diff
@@ -305,7 +324,9 @@ Provide the master embedded journal addresses and set the hostname of the curren
 ```console
 $ docker run -d \
   ...
-  -e ALLUXIO_JAVA_OPTS="-Dalluxio.master.embedded.journal.addresses=master-hostname-1:19200,master-hostname-2:19200,master-hostname-3:19200 -Dalluxio.master.hostname=master-hostname-1" \
+  -e ALLUXIO_JAVA_OPTS=" \
```
[the remark this answers is not in the document]
Comment (apc999) [reply docs/en/deploy/Running-Alluxio-On-Docker.md:327]: yes, I tested all commands here

File: docs/en/deploy/Running-Alluxio-On-Docker.md [hunk 2]
```diff
@@ -330,7 +353,10 @@ Point them to a shared journal and set their Zookeeper configuration.
 ```console
 $ docker run -d \
   ...
-  -e ALLUXIO_JAVA_OPTS="-Dalluxio.master.journal.type=UFS -Dalluxio.master.journal.folder=hdfs://[namenodeserver]:[namenodeport]/alluxio_journal -Dalluxio.zookeeper.enabled=true -Dalluxio.zookeeper.address=zkhost1:2181,zkhost2:2181,zkhost3:2181" \
+  -e ALLUXIO_JAVA_OPTS="-Dalluxio.master.journal.type=UFS \
```
[the remark this answers is not in the document]
Comment (apc999) [reply docs/en/deploy/Running-Alluxio-On-Docker.md:356]: done

File: docs/en/deploy/Running-Alluxio-On-Docker.md [hunk 3]
```diff
@@ -296,7 +314,8 @@ $ docker exec ${container_id} cat /opt/alluxio/conf/alluxio-env.sh
 A lone Alluxio master is a single point of failure. To guard against this, a production
 cluster should run multiple Alluxio masters in [High Availability mode]({{ '/en/deploy/Running-Alluxio-On-a-HA-Cluster.html' | relativize_url }}).
 
-#### Option A: Internal Leader Election
```
[the remark this answers is not in the document]
Comment (apc999) [reply docs/en/deploy/Running-Alluxio-On-Docker.md:299]: done

**Comment by apc999** [comment]
@Xenorith  Thanks for the comments. Please take a look updated version

File: docs/en/deploy/Running-Alluxio-On-Docker.md [hunk 4]
```diff
@@ -296,7 +314,12 @@ $ docker exec ${container_id} cat /opt/alluxio/conf/alluxio-env.sh
...
+There are two ways to enable HA mode in Alluxio, either with internal leader election and embedded journal, or external Zookeeper and a shared journal storage.
+Please read [running Alluxio with HA]({{ '/en/deploy/Running-Alluxio-On-a-HA-Cluster.htmll#overview' | relativize_url }}) for more details.
```
Comment (Xenorith) [inline docs/en/deploy/Running-Alluxio-On-Docker.md:318]: jenkins caught a typo:
extra l in -> `.htmll#overview`

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Comment by apc999** [comment]
alluxio-bot, cherry-pick this to branch-2.3 please

**Pull request merged**